src/gc/gc.h

```diff
@@ -352,19 +197,7 @@ void record_changed_seg (uint8_t* start, uint8_t* end,
 void record_global_mechanism (int mech_index);
 #endif //GC_CONFIG_DRIVEN
 
-//constants for the flags parameter to the gc call back
-
-#define GC_CALL_INTERIOR            0x1
-#define GC_CALL_PINNED              0x2
-#define GC_CALL_CHECK_APP_DOMAIN    0x4
-
-//flags for GCHeap::Alloc(...)
-#define GC_ALLOC_FINALIZE 0x1
-#define GC_ALLOC_CONTAINS_REF 0x2
-#define GC_ALLOC_ALIGN8_BIAS 0x4
-#define GC_ALLOC_ALIGN8 0x8
-
-class GCHeap {
+class GCHeap : public IGCHeap {
```

Comment:
> They can be on IGCHeapInternal interface that is internal implementation detail of the GC.

This would seem to be the purpose that `::GCHeap` fulfills today. Would you agree with renaming `GCHeap` to `IGCHeapInternal`, in that case? Or leaving as it is now?

I'll make the change re: the VM and GC statics.